explorer/explorerroutes.go

```diff
@@ -30,6 +30,35 @@ import (
 	humanize "github.com/dustin/go-humanize"
 )
 
+// links to be passed with common page data.
+type links struct {
+	CoinbaseComment string
+	POSExplanation  string
+	TestNet         string
+	Explorer        string
+	APIDocs         string
+	Github          string
+	License         string
+	NetParams       string
+	TestNetSearch   string
+	ExplorerSearch  string
+	BtcAddress      string
+}
+
+var explorerLinks = &links{
+	CoinbaseComment: "https://github.com/decred/dcrd/blob/2a18beb4d56fe59d614a7309308d84891a0cba96/chaincfg/genesis.go#L17-L53",
+	POSExplanation:  "https://docs.decred.org/faq/proof-of-stake/general/#9-what-is-proof-of-stake-voting",
+	TestNet:         "http://testnet.dcrdata.org/",
+	Explorer:        "https://explorer.dcrdata.org/",
```

Comment:
This one will need to be settable by config so the dcrdata url base can be changed for different deployments.